### PR TITLE
Assign / Remove Supervisor by Supervisor

### PR DESCRIPTION
**Pull request by abdullahallmehedi**

### CODE REVIEW

**Comment by rofiuddin** [comment]
okay

File: app/helpers/supervisor/users_courses_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module Supervisor::UsersCoursesHelper
```
Comment (dieunb) [inline app/helpers/supervisor/users_courses_helper.rb]: remove this file if you don't use it

File: app/models/users_course.rb [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+  ASSIGN_TYPE = {
+    supervisor: "supervisor",
+    trainee: "trainee"
+  }
+
   belongs_to :user
   belongs_to :course
-end
+
+  scope :supervisors, ->{ where(supervisor: true) }
```
Comment (dieunb) [inline app/models/users_course.rb]: convention, should `{where supervisor: true}`

File: app/models/users_course.rb [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+  scope :trainees, ->{ where(supervisor: false) }
```
Comment (dieunb) [inline app/models/users_course.rb]: `{where supervisor: false}`

**Pull request merged**